src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+         class="card__img"
+         alt="imac"
+         src="images/imac.jpeg"
+       >
+
+      <div class="card__link">
+        <a>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</a>
+      </div>
+
+      <div class="card__code">
+        Product code: 195434
+      </div>
+
+      <div class="card__grade">
+        <div class="card__grade-stars">
+          <div class="stars-4">
```

Comment:
Why stars-4? Here should be the stars block from the previous task with the possibility to change the active stars number
<img width="938" alt="image" src="https://user-images.githubusercontent.com/99403198/219865349-eb47cd62-7029-4a04-b380-c05cd6d83f5b.png">

```suggestion
          <div class="stars stars--4">
```